src/test/java/spec/yandexSpec/CreateAccount.java

```diff
@@ -18,46 +18,49 @@
 
 public class CreateAccount extends BaseSpec {
     private static final Logger logger = LoggerFactory.getLogger(CreateAccount.class);
-    private static final String EXPECTED_TITLE = "Inbox () — Yandex.Mail";
 
     private HomePage homePage;
     private LogInPage logInPage;
     private MailPage mailPage;
 
     @Test
     public void verifyYandexHomePage() {
-        String ExpectedPageTitle = "Yandex.Mail — free, reliable email";
+        homePage = new HomePage(driver);
 
-        Assertions.assertEquals(driver.getTitle(), ExpectedPageTitle);
+        Assertions.assertTrue(driver.getTitle().contains(ProjectVariables.EXPECTED_TITLE));
     }
 
     @Test
-    public void verifyUserCanLogInSuccessfully() throws InterruptedException {
+    public void verifyUserCanLogInSuccessfully() {
         homePage = new HomePage(driver);
         logInPage = homePage.clickOnLogInButton();
         mailPage = logInPage.logIn(ProjectVariables.PHONE_NUMBER, ProjectVariables.PASSWORD);
 
-        //Sleep is neither implicit nor explicit wait. It is way of controlling the test flow
-        //by freezing the code execution for certain amount of time
-        Thread.sleep(1000);
-
         mailPage.switchToLightVersion();
         WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(3));
         wait.pollingEvery(Duration.ofSeconds(1));
         wait.until(ExpectedConditions.presenceOfElementLocated(mailPage.getCUSTOMER_NAME()));
 
-        Assertions.assertEquals(driver.getTitle(), EXPECTED_TITLE);
+        Assertions.assertTrue(driver.getTitle().contains(ProjectVariables.EXPECTED_TITLE));
         logger.info("Yandex Logo is not displayed");
```

Comment:
Do not put any code after assertion, because it will be not executed if assertion fail

Reply:
removed